student.rb

```diff
@@ -2,9 +2,12 @@
 require_relative 'person'
 
 class Student < Person
```

Comment:
- [ ] I see you've not updated your Person class so I'll leave the suggestion here. Please make sure to add the necessary associations between the Person class and the Rental class as you did in the Book class. 🙏 
- [ ] Please make sure to create the `add_rental` method in the Person class properly. 👍